src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -117,21 +117,30 @@ DlgPrefRecord::DlgPrefRecord(QWidget* parent, UserSettingsPointer pConfig)
 
     slotApply();
     // Make sure a corrupt config file won't cause us to record constantly.
+    // TODO: What does this do exactly? I thougth it stopped the recording when showing
+    // the preferences page but it does not (and i would preffer that keeps not stopping it).
     m_pRecordControl->set(RECORD_OFF);
```

Comment:
All control objects are 0.0 = RECORD_OFF by default so the whole block with comments can be removed.